### PR TITLE
Fix: filter out edge-app-files from assets list command

### PR DESCRIPTION
**Pull request by rusko124**
## What does this PR do?
filter out edge-app-files from assets list command. Just the same way as we do it in web-console.

## GitHub issue or Phabricator ticket number?
https://phorge.wireload.net/T7310

## How has this been tested?
Manually

## Checklist before merging

- [ ] If have added tests.
- [ ] Is this a new feature? If yes, please write one phrase about this update.
- [ ] I've attached relevant screenshots (if relevant).

### CODE REVIEW

File: src/commands/asset.rs [hunk 1]
```diff
@@ -23,7 +23,7 @@ impl AssetCommand {
     pub fn list(&self) -> anyhow::Result<Assets, CommandError> {
         Ok(Assets::new(commands::get(
             &self.authentication,
-            "v4/assets",
+            "v4/assets?type=in.('appweb','audio','edge-app','image','video','web')",
```
Comment (korvyashka) [inline src/commands/asset.rs:26]: lets remove appweb - they are not used

**Review by korvyashka — APPROVED** [review]

File: src/commands/asset.rs (hunk 1 above)
Comment (sergey-borovkov) [inline src/commands/asset.rs:26]: Why not say everything but edge app files ? If we add new asset type it won't be shown by default

Comment (rusko124) [reply src/commands/asset.rs:26]: > If we add new asset type it won't be shown by default

Yes, we want exactly this approach.

File: src/commands/asset.rs [hunk 2]
```diff
@@ -214,6 +214,7 @@ mod tests {
         mock_server.mock(|when, then| {
             when.method(GET)
                 .path("/v4/assets")
+                .query_param("type", "in.('appweb','audio','edge-app','image','video','web')")
```
Comment (rusko124) [inline src/commands/asset.rs:217]:
```suggestion
                .query_param("type", "in.('audio','edge-app','image','video','web')")
```

File: src/commands/asset.rs (hunk 1 above)
Comment (rusko124) [inline src/commands/asset.rs:26]:
```suggestion
            "v4/assets?type=in.('audio','edge-app','image','video','web')",
```

Comment (korvyashka) [reply src/commands/asset.rs]: That should be deleted from appserver too - let's not increase tech debt.

Comment (vpetersson) [reply src/commands/asset.rs]: We should remove `appweb`. But really, we should filter out edge-app files in the API rather than as a filter.

**Pull request merged**